lms/templates/fields/field_dropdown_account.underscore

```diff
@@ -20,6 +20,13 @@
         <span class="u-field-value-readonly"></span>
     <% } else { %>
         <select name="select" id="u-field-select-<%- id %>" aria-describedby="u-field-help-message-<%- id %>">
+            <% if (preOptions) { %>
+                <optgroup label="Common Time Zones">
+                <% _.each(preOptions, function(time_zone) { %>
+                   <option value="<%- time_zone.time_zone %>"><%- time_zone.description %></option>
+                <% }); %>
+                <optgroup label="All Time Zones">
```

Comment:
Everything else in this template is very generalized - could we do the same for preOptions? You may want to then pass the "beforePre" and "afterPre" label strings as parameters to the template - which also gives us the chance to translate them (using `gettext` in javascript).